docs/dev_guide/newcomers.rst

```diff
@@ -37,8 +37,9 @@ Code
 If you would prefer to code Python instead, we have a SunPy issue list on Github `issues`_ where all the known issues with SunPy are kept.
 An issue might have a label and the `Package Novice label`_ is a good place to start.
 These are issues that have been deemed a good way to be eased into SunPy and are achievable with little understanding of the SunPy codebase.
-If you came here from the Mozilla Sprint, we have a `mozsprint label.`_
-These issues are similar in nature to the Package Novice label but with a slight difference that these issues are (we think) achievable within the timescale of a sprint, typically one day of hacking.
+We also tag issues for specific events such as the Mozilla Sprint
+(`mozsprint label.`_) or hacktoberfest. The scope of the issues should be appropriate for
```

Comment:
@nabobalis is this actually still true?

Reply:
I thought we merged them into one label. I could be wrong.

Reply:
Yeah we don't have the `mozsprint` label anymore, I think we decided to cover it under `Hacktoberfest `. I really don't several labels for the same kind of event.